RELEASE.md

```diff
@@ -39,7 +39,9 @@ Release cadence of first pre-releases being cut is 6 weeks.
 | v2.32          | 2021-12-01                                 | Julius Volz (GitHub: @juliusv)              |
 | v2.33          | 2022-01-12                                 | Björn Rabenstein (GitHub: @beorn7)          |
 | v2.34          | 2022-02-23                                 | Chris Marchbanks (GitHub: @csmarchbanks)    |
-| v2.35          | 2022-04-06                                 | **searching for volunteer**                 |
+| v2.35          | 2022-04-06                                 | Augustin Husson (GitHub: @nexucis)          |
+| v2.36          | 2022-05-18                                 | Matthias Rampke (GitHub: @metalmatze        |
```

Comment:
```suggestion
| v2.36          | 2022-05-18                                 | Matthias Loibl (GitHub: @metalmatze        |
```
Confusion, there are two Matthias now. :smile:

Reply:
Ah sorry @metalmatze. @roidelapluie just mentionned Matthias. So I had 50% chance to put the good one haha